### ISSUE TITLE
node-gyp build error

### ISSUE THREAD

**Issue by elafans** [elastos/Elastos.NET.Carrier.Nodejs.SDK#36]
An error occurred while building node-gyp.(directory error)

What should i do??

![error](https://user-images.githubusercontent.com/39670642/40585483-6ee6827a-61ee-11e8-880a-df34ab8f6555.PNG)

**Comment by kuit** [comment]
Do you build carrier SDK? And set CARRIER_NATIVE_SDK environment value before "node-gyp configure" command?

**Comment by madhorseELA** [comment]
YES I did. i build carrier_native_sdk. i did it all you said. TT

**Comment by kuit** [comment]
Can do this?
$ ls $CARRIER_NATIVE_SDK/include/ela_carrier.h

**Comment by madhorseELA** [comment]
The cause of the problem is estimated that i built all in administrator mode. I cleaned everything related and built in normal mode, so it worked fine.

Thank you for your help.

![done](https://user-images.githubusercontent.com/39670642/40588280-1af5e318-6216-11e8-930f-34352dc5a922.PNG)
